doc/source/features/pxc-strict-mode.rst

```diff
@@ -303,7 +305,7 @@ Depending on the selected mode, the following happens:
 Explicit table locking
 ----------------------
 
-|PXC| has only experimental support for explicit table locking operations,
+|PXC| supports explicit table locking operations,
```

Comment:
This is suspect. AFAIK PXC does not correctly and fully support explicit table locks in that the locks are not propagated and taken on all nodes, only the node which received the lock request.  This may lead to unexpected certification failures later on that leads to a different observable behavior of a cluster vs a non-cluster instance of PS.

Reply:
Changed to tech preview support